guides/micronaut-mongodb-synchronous/java/src/main/java/example/micronaut/FruitController.java

```diff
@@ -0,0 +1,37 @@
+package example.micronaut;
+
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.Post;
+import io.micronaut.http.annotation.Status;
+import io.micronaut.scheduling.TaskExecutors;
+import io.micronaut.scheduling.annotation.ExecuteOn;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@Controller("/fruits") // <1>
+public class FruitController {
+
+    private final FruitRepository fruitService;
+
+    public FruitController(FruitRepository fruitService) {  // <2>
+        this.fruitService = fruitService;
+    }
+
+    @ExecuteOn(TaskExecutors.IO)  // <3>
```

Comment:
should this be at the class level instead of repeated for both methods?

Reply:
I prefer to have it in a method level because it is easy to refactor. You cut the method  and extract it for example to another controller and you don't leave any annotation behind.  It is a personal perference. 

I have moved it to a class level if that it is what we want to do.